AuthenticatorShared/Core/Platform/Services/Stores/AppSettingsStore.swift

```diff
@@ -392,6 +403,12 @@ extension DefaultAppSettingsStore: AppSettingsStore {
         fetch(for: .debugFeatureFlag(name: name))
     }
 
+    func hasSyncedAccount(name: String) -> Bool {
+        let hashedData = SHA256.hash(data: Data(name.utf8))
+        let hashedName = hashedData.map { String(format: "%02hhx", $0) }.joined()
+        return fetch(for: .hasSyncedAccount(name: hashedName))
+    }
```

Comment:
🤔 Not sure if this is missing a merge from `main` but now there's a `String` extension that gives you the hashed SHA256 directly, called [hexSHA256Hash](https://github.com/bitwarden/authenticator-ios/blob/main/AuthenticatorShared/UI/Platform/Application/Extensions/String.swift#L50)